aws/cloudformation-templates/web-ui-pipeline.yaml

```diff
@@ -139,7 +139,12 @@ Resources:
 
                 for obj in source_bucket.objects.filter(Prefix=source_path):
                     source = { 'Bucket': source_bucket_name, 'Key': obj.key }
-                    target = target_bucket.Object(obj.key)
+                    if source_path != "images":
```

Comment:
Would it be a better check to remove the relative path exists in the source path rather than checking if it the string "images"?
This would then work even if the "images" folder was renamed/restructured

Reply:
that's a very good suggestion